### PR TITLE
Update ERC-4337: Extract aggregation to a separate ERC

### PR DESCRIPTION
**Pull request by forshtat**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-4337.md [hunk 1]
```diff
@@ -132,18 +119,15 @@ The `userOpHash` is a hash over the userOp (except signature), entryPoint and ch
...
 
 * MUST validate the caller is a trusted EntryPoint
-* If the account does not support signature aggregation, it MUST validate that the signature is a valid signature of the `userOpHash`, and
+* MUST validate that the signature is a valid signature of the `userOpHash`, and
   SHOULD return SIG_VALIDATION_FAILED (and not revert) on signature mismatch. Any other error MUST revert.
 * MUST pay the entryPoint (caller) at least the "missingAccountFunds" (which might be zero, in case the current account's deposit is high enough)
 * The account MAY pay more than this minimum, to cover future transactions (it can always issue `withdrawTo` to retrieve it)
 * The return value MUST be packed of `authorizer`, `validUntil` and `validAfter` timestamps.
-  * authorizer - 0 for valid signature, 1 to mark signature failure. Otherwise, an address of an authorizer contract. This ERC defines a "signature aggregator" as an authorizer.
+  * authorizer - 0 for valid signature, 1 to mark signature failure. Otherwise, an address of an authorizer contract, as defined in [ERC-XXXX](link).
```
Comment (shahafn) [inline ERCS/erc-4337.md:127]: TODO fix link when assigned numbers

Comment (forshtat) [reply ERCS/erc-4337.md]: Fixed. Waiting for ERC-7766 to be merged.

File: ERCS/erc-4337.md [hunk 2]
```diff
@@ -390,19 +344,15 @@ struct ReturnInfo {
...
-
 struct StakeInfo {
   uint256 stake;
   uint256 unstakeDelaySec;
 }
 
-
 ```
 
+The `AggregatorStakeInfo` structure is further defined in [ERC-XXXX](link).
```
[the remark this answers is not in the document]
Comment (forshtat) [reply ERCS/erc-4337.md]: Fixed. Waiting for ERC-7766 to be merged.

**Comment by SamWilsn** [comment]
Close then reopen this after the other PR is merged and this one will go through.

**Pull request merged**